google/cloud/internal/log_impl_test.cc

```diff
@@ -68,62 +68,85 @@ TEST(DefaultLogBackend, CircularBuffer) {
   ScopedEnvironment config(kLogConfig, "lastN,5,WARNING");
   ScopedEnvironment clog(kEnableClog, absl::nullopt);
   auto be = DefaultLogBackend();
-  EXPECT_TRUE(!!be);
   auto const* buffer = dynamic_cast<CircularBufferBackend*>(be.get());
   ASSERT_NE(buffer, nullptr);
   EXPECT_EQ(5, buffer->size());
   EXPECT_EQ(Severity::GCP_LS_WARNING, buffer->min_flush_severity());
+  auto const* clog_be = dynamic_cast<StdClogBackend*>(buffer->backend().get());
+  ASSERT_NE(clog_be, nullptr);
+  EXPECT_EQ(Severity::GCP_LS_DEBUG, clog_be->min_severity());
 }
 
 TEST(DefaultLogBackend, CLog) {
   ScopedEnvironment config(kLogConfig, "clog");
   ScopedEnvironment clog(kEnableClog, absl::nullopt);
   auto be = DefaultLogBackend();
-  EXPECT_TRUE(!!be);
-  EXPECT_NE(dynamic_cast<StdClogBackend*>(be.get()), nullptr);
+  auto const* clog_be = dynamic_cast<StdClogBackend*>(be.get());
+  ASSERT_NE(clog_be, nullptr);
```

Comment:
nit: here and below, consider `ASSERT_THAT(clog_be, IsNotNull());`

Reply:
And the one above too.

Reply:
Done.